subcell_pipeline/simulation/cytosim/run_cytosim_compression_batch_simulations.py

```diff
@@ -0,0 +1,171 @@
+# %% [markdown]
+"""
+# Run Cytosim compression simulations
+
+Notebook contains steps for running Cytosim simulations in which a single actin
+fiber is compressed at different compression velocities.
+
+This notebook provides an example of running a simulation series in which there
+are multiple conditions, each of which need to be run for multiple replicates.
+For an example of running a simulation series for a single condition for
+multiple replicates, see `run_cytosim_single_fiber_batch_simulations.py`.
+
+- [Define simulation conditions](#define-simulation-conditions)
+- [Generate configs from template](#generate-configs-from-template)
+- [Define simulation settings](#define-simulation-settings)
+- [Create and run jobs](#create-and-run-jobs)
+"""  # noqa: D400, D415
+
+# %%
+import getpass
+import sys
+from datetime import datetime
+
+from subcell_pipeline.simulation.run_batch_simulations import (
+    check_and_save_job_logs,
+    generate_configs_from_template,
+    register_and_run_simulations,
+)
+
+# %% [markdown]
+"""
+Template generator uses the Preconfig class from Cytosim. Append the path to
+`preconfig.py` from a copy of the `Cytosim` repository, or download a copy into
+this location.
+"""
+
+# %%
+sys.path.append("/home/jessica/Documents/Cytosim/python/run/")
+from preconfig import Preconfig  # noqa: E402
+
+# %% [markdown]
+"""
+## Define simulation conditions
+
+Defines the `COMPRESSION_VELOCITY` simulation series, which compresses a single
+500 nm actin fiber at four different velocities (4.7, 15, 47, and 150 $\mu m$/s)
+with five replicates each (random seeds 1, 2, 3, 4, and 5).
+"""
+
+# %%
+# Name of the simulation series
+series_name: str = "COMPRESSION_VELOCITY"
+
+# S3 bucket for input and output files
+bucket: str = "s3://cytosim-working-bucket"
+
+# Random seeds for simulations
+random_seeds: list[int] = [1, 2, 3, 4, 5]
+
+# Path to the config template file
+path_to_template: str = "../../../templates/vary_compress_rate.cym.tpl"
```

Comment:
nit: could use `pathlib` here for OS agnostic separators

```suggestion
from pathlib import Path
path_to_template: str = Path(__file__).parents[3] / "templates/vary_compress_rate.cym.tpl"
```